src/my_plugin/my_plugin_android/android/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="com.example.my_plugin">
```

Comment:
What's the reason for dropping the package attribute here?

Reply:
That is causing a warning with AGP 8.2.2. That is migrated to namespace in the build.gradle file